src/journey-maps/angular/components/level-switch/services/map-layer-filter.ts

```diff
@@ -31,20 +31,23 @@ export class SbbMapLayerFilter {
     this._setLayerVisibility('rokas_background_mask', level < 0);
   }
 
-  private _calculateLayerFilter(oldFilter: any[], level: number): any[] {
+  private _calculateLayerFilter(
+    oldFilter: FilterSpecification | void,
```

Comment:
void should only be used as a return value
Use undefined, if the value might be undefined